frontend/ContextMenu.js

```diff
@@ -58,16 +63,39 @@ class ContextMenu extends React.Component {
     this.props.hideContextMenu();
   }
 
+  getHeight = element => {
```

Comment:
No worries. I assume you just branched from your other branch instead of master, maybe accidentally it sounds like 😁